module/zfs/lz4.c

```diff
@@ -87,7 +87,7 @@ lz4_decompress_zfs(void *s_start, void *d_start, size_t s_len,
 
 	/*
 	 * Returns 0 on success (decompression function returned non-negative)
-	 * and non-zero on failure (decompression function returned negative.
+	 * and non-zero on failure (decompression function returned negative.)
```

Comment:
Oops, thanks.

PS: I got my buildbot running, so I'm hoping to have some of the last few bugs ironed out soon.